src/components/Todo/Todo.tsx

```diff
@@ -22,20 +21,25 @@ export const Todo: FC<Props> = ({ todo }) => {
       role="menuitem"
       tabIndex={Math.random()}
     >
-      <p className={css(styles.id, typography.altBody1)}>{todo.id}</p>
+      <div className={css(styles.id, typography.altBody1)}>
```

Comment:
I tried to find a way to set paragraph like a div, but I didn't find any good way to do that.